integration-tests/bridges/environments/polkadot-kusama/bridges_polkadot_kusama.sh

```diff
@@ -133,7 +133,7 @@ ON_BRIDGE_HUB_KUSAMA_SOVEREIGN_ACCOUNT_FOR_LANE_00000001_bhpd_ThisChain="EoQBtnw
 ON_BRIDGE_HUB_KUSAMA_SOVEREIGN_ACCOUNT_FOR_LANE_00000001_bhpd_BridgedChain="EoQBtnwp4jMtCEpV7CPsssT6bdDHuHZmf3aGXxHJiSA4Dz3"
```

Comment:
When will these tests be integrated into CI? :P

Reply:
Good question! I think the main issue is the `sudo` tweaking mentioned in https://github.com/paritytech/parity-bridges-common/issues/3016, but it should be solvable. Additionally, there are open issues for adding Chopsticks support.  

I created a tracking issue for this: https://github.com/polkadot-fellows/runtimes/issues/544.

Reply:
Sudo can just be worked around by doing on chain proposal and fast track it with a fake fellowship :D For sure a little bit more complicated, but doable with chopsticks.